src/Tabs.js

```
@@ -1,43 +1,74 @@
 import React from 'react';
 import cx from 'classnames';
+import idgen from './idgen';
 
 import Row from './Row';
 import Col from './Col';
 
 class Tabs extends React.Component {
+  componentDidMount() {
+    if (typeof $ !== 'undefined') {
```

Would need to Throw an exception